jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js

```diff
@@ -101,6 +101,22 @@ class ProfileBundleList extends React.Component {
       </h2>);
   }
 
+  /**
+    Sort by the key attribute.
+    @param {Object} a The first value.
+    @param {Object} b The second value.
+    @returns {Array} The sorted array.
+  */
+  _byName(a, b) {
```

Comment:
Can this be moved into the BasicTable? The sort key could then take a string or a function?

Reply:
This is the existing way that the table handles sorting.